### ISSUE TITLE
Option to strip underscore prefix with DataProviderNameFixer

### ISSUE THREAD

**Issue by weshooper** [kubawerlos/php-cs-fixer-custom-fixers#337]
Is it possible to exclude the underscore when the prefix is empty?

Config:

`['prefix' => '', 'suffix' => '_data_provider']`

Outcome:

```diff
<?php
 class FooTest extends TestCase {
     /**
-     * @dataProvider dataProvider
+     * @dataProvider happy_path_data_provider
      */
     public function test_happy_path() {}
-    public function dataProvider() {}
+    public function _happy_path_data_provider() {}
 }
```

Expected:

```diff
<?php
 class FooTest extends TestCase {
     /**
-     * @dataProvider dataProvider
+     * @dataProvider happy_path_data_provider
      */
     public function test_happy_path() {}
-    public function dataProvider() {}
+    public function happy_path_data_provider() {}
 }
```

**Comment by kubawerlos** [comment]
@weshooper it was released with `2.3.0`.

**Comment by weshooper** [comment]
@kubawerlos thank you so much! That looks like a nice release :-)
